app/services/quotable_service.rb

```diff
@@ -0,0 +1,18 @@
+require 'json'
+
+class QuotableService
+  def self.conn 
+    Faraday.new(
+      url: 'https://api.quotable.io'
+    )
+  end
+
+  def self.quote_search_endpoint(keyword)
+    response = conn.get('/search/quotes', { query: keyword })
```

Comment:
One other thing that you could probably do here at the service call is limit it to 10. Then you won't have to worry about doing that logic elsewhere.